client/src/components/Creator.js

```diff
@@ -1,8 +1,195 @@
+      let web3 = new Web3(this.props.provider);
+
+      // The minimum ABI to get ERC20 Token balance
+      let minABI = [
+        // balanceOf
+        {
+          constant: true,
+          inputs: [{ name: "_owner", type: "address" }],
+          name: "balanceOf",
+          outputs: [{ name: "balance", type: "uint256" }],
+          type: "function",
+        },
+        // decimals
+        {
+          constant: true,
+          inputs: [],
+          name: "decimals",
+          outputs: [{ name: "", type: "uint8" }],
+          type: "function",
+        },
+      ];
+
+      // Get ERC20 Token contract instance
+      var contract = new web3.eth.Contract(minABI, tokenAddress);
+      // Call balanceOf function
+      var balance = await contract.methods.balanceOf(walletAddress).call();
+      this.setState({ aUSDCBalance: balance });
+    }
+  }
+
+  componentDidMount() {
+    this.getBalance();
+    fetch(
+      "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&ids=ethereum"
+    )
+      .then((response) => response.json())
+      .then((priceUSD) => {
+        this.setState({ ETHinUSD: priceUSD[0].current_price });
+      });
+  }
+
+  getValue(weiVal) {
+    const val = Web3.utils.fromWei(weiVal, "ether");
+
+    var USD = parseFloat(val) * parseFloat(this.state.ETHinUSD);
+    USD = USD.toFixed(2);
+    return val + " ETH ($" + USD + " USD)";
+  }
+
+  getRealTime(timestamp) {
+    var utcSeconds = parseInt(timestamp);
+    var d = new Date(0); // The 0 there is the key, which sets the date to the epoch
+    d.setUTCSeconds(utcSeconds);
+    return d.toLocaleString();
+  }
+
   render() {
-    return <div>Creator page</div>;
+    const balance = this.state.aUSDCBalance + " aUSDC";
+    const fanLink =
+      "https://blossym.org/fan?creatorAddress=" + this.props.connectedWallet;
+    let ethList;
+    if (this.props.ethTransactions) {
+      ethList = (
+        <Container>
+          {this.props.ethTransactions.map((tx) => (
+            <tr>
+              <td>{tx.from}</td>
+              <td>{this.getValue(tx.value)}</td>
+              <td>{this.getRealTime(tx.timeStamp)}</td>
+            </tr>
+          ))}
+        </Container>
+      );
+    } else {
+      ethList = <h3> Hasn't loaded yet! </h3>;
+    }
+    let page;
+    if (!this.props.connectedWallet) {
+      page = (
+        <div>
+          <div class="d-flex justify-content-center mt-5 col-md-12">
+            <Button
+              variant="primary"
+              onClick={this.props.onWalletConnectClick}
+              data-testid="connect-wallet-button"
+            >
+              Connect Wallet
+            </Button>
+          </div>
+        </div>
+      );
+    } else {
+      page = (
+        <div>
+          <div class="d-flex justify-content-center mt-5 col-md-12">
+            <Button variant="outline-success">Cash Out</Button>
+          </div>
+
+          <div class="d-flex justify-content-around mt-5 col-md-12">
```

Comment:
Not necessary for this PR, but this div will not allow the cards to rearrange themselves when the screen width is less wide, e.g. on mobile.

To do so, instead of setting custom widths for each card (the cards should be automatically sized based on space):

1.  Replace this div with a `<Container>`
2.  Add a `<Row`> inside it
3.  Wrap each card in a `<Col sm>` within the `<Row>` (`sm` will indicate that they columns should stack as single column when the width of the container is <576px)

This'll allow the cards to stack on top of each other when the browser width is smaller instead of staying in a row even on mobile. More details: https://getbootstrap.com/docs/4.0/layout/grid/